comms/.air.toml

```diff
@@ -7,7 +7,7 @@ tmp_dir = "/tmp"
   bin = "/tmp/main"
   cmd = "go build -o /tmp/main ."
   delay = 0
-  exclude_dir = ["/nats", "assets", "tmp", "vendor", "testdata", "storage/storageserver/weather-map"]
```

Comment:
I filtered out all of `weather-map/` except `weather-map/dist/` via .dockerignore, so this is not needed. Dist is needed to host reload